libs/server-sdk/src/client_impl.cpp

```diff
@@ -200,7 +203,21 @@ template <typename T>
 EvaluationDetail<T> ClientImpl::VariationInternal(Context const& ctx,
```

Comment:
All the grossness in this method that follows is fixed in a future PR (https://github.com/launchdarkly/cpp-sdks/pull/203)